test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -13,10 +13,15 @@ Describe "Get-Process for admin" -Tags @('CI', 'RequireAdminOnWindows') {
         }
     }
 
-    It "Should support -FileVersionInfo" -Skip:(!$IsWindows) {
+    It "Should support -FileVersionInfo" {
         $pwshVersion = Get-Process -Id $pid -FileVersionInfo
         $PSVersionTable.PSVersion | Should -Match $pwshVersion.FileVersion
     }
+
+    It "Should run correctly with parameter -FileVersionInfo also when process' main module is null." {
+        # Main module for idle process can be null on non-Windows platforms
+        { $pwshVersion = Get-Process -Id 0 -FileVersionInfo } | Should -Not -Throw
```

Comment:
There's two tests here, one for Windows, one for non-Windows.  For Windows, we need to add `-ErrorAction Stop` and validate the error on the Idle process.  For non-Windows, we should still add `-ErrorAction Stop` but just check it didn't throw

Reply:
Added 2 tests.